kafka-cron/prometheus.yml

```diff
@@ -0,0 +1,42 @@
+## prometheus.yml ##
+
+# global settings
```

Comment:
For any metrics that you are displaying in your dashboards or where you're using them for alerting rules, you should use recording rules: https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/
Otherwise our dashboards can get very slow in a real production context. 

You don't have to do it for everything, but please setting up at least one recording rule and using it from a dashboard or alert.